### ISSUE TITLE
[Bug Bash][Accessibility] There is no horizontal scroll bar at the bottom of PMC window

### ISSUE THREAD

**Issue by v-luzh** [NuGet/Home#7902]
## Details about Problem
NuGet version: NuGet Client Dev\5.1.0.5906
VS Version: D16.1stg\28721.190
OS: Windows 10 RS4_Release 17134.1.180410-1804
## Detailed repro steps
1. Create a Console App (.NET Framework) in VS.
2. Open PMC through clicking menu Tools-> NuGet Package Manager ->Package Manager Console.
3. Start shrinking width (to the left) of the VS Internal window.
## Expected
There should be a horizontal scroll bar at the bottom of PMC window (should be same with Output window).
## Actual
There is no horizontal scroll bar at the bottom of PMC window as below screenshot.
![NoHorizontalScrollBar](https://user-images.githubusercontent.com/20632625/54815461-01941780-4ccd-11e9-814d-c60f93e22b11.png)
## Note
It is not a regression issue. It also repro on D16.0 28625.133 (NuGet 5.0.0.5856), D15.9 28307.158 (NuGet 4.9.3.5777) and D15.8 28010.2050 (NuGet 4.8.1.5435).

**Comment by nkolev92** [comment]
Why is this a bug though? 

The PMC console is responsive. I'd argue it's even better than the output window. 

@rrelyea thoughts on this one? 
Personally I think this is a good behavior.

**Comment by v-luzh** [comment]
We thought there should not be arrows to the left and to the right at the bottom of PMC if we decide not to show the horizontal scroll bar.

**Comment by v-luzh** [comment]
It still reproes on VS Main\35221.199 + NuGet Client Dev\6.12.0.81.

**Comment by nkolev92** [comment]
Low priority, won't fix.
